modules/mobile/spec/request/letters_request_spec.rb

```diff
@@ -3,139 +3,92 @@
+              },
+              {
+                'name' => 'Benefit Verification Letter',
+                'letterType' => 'benefit_verification'
+              }
+            ]
         }
       }
     }
   end
 
+  before do
+    token = 'abcdefghijklmnop'
+    allow_any_instance_of(Lighthouse::LettersGenerator::Configuration).to receive(:get_access_token).and_return(token)
+    allow(File).to receive(:read).and_return(rsa_key.to_s)
+    allow_any_instance_of(IAMUser).to receive(:icn).and_return('24811694708759028')
+    user = build(:iam_user)
+    iam_sign_in(user)
+    Flipper.enable(:mobile_lighthouse_letters, user)
+  end
+
   describe 'GET /mobile/v0/letters' do
-    context 'with a valid evss response' do
+    context 'with a valid lighthouse response' do
       it 'matches the letters schema' do
-        VCR.use_cassette('evss/letters/letters') do
+        VCR.use_cassette('mobile/lighthouse_letters/letters_200', match_requests_on: %i[method uri]) do
           get '/mobile/v0/letters', headers: iam_headers
           expect(response).to have_http_status(:ok)
           expect(JSON.parse(response.body)).to eq(letters_body)
-          expect(response.body).to match_json_schema('letters')
-        end
-      end
-    end
-
-    unauthorized_five_hundred = { cassette_name: 'evss/letters/unauthorized' }
-    context 'with an 500 unauthorized response', vcr: unauthorized_five_hundred do
-      it 'returns a bad gateway response' do
-        get '/mobile/v0/letters', headers: iam_headers
-        expect(response).to have_http_status(:bad_gateway)
-        expect(response.body).to match_json_schema('evss_errors')
-      end
-    end
-
-    context 'with a 403 response' do
-      it 'returns a not authorized response' do
-        VCR.use_cassette('evss/letters/letters_403') do
-          get '/mobile/v0/letters', headers: iam_headers
-          expect(response).to have_http_status(:forbidden)
-          expect(response.body).to match_json_schema('evss_errors')
-        end
-      end
-    end
 
-    context 'with a generic 500 response' do
-      it 'returns a not found response' do
-        VCR.use_cassette('evss/letters/letters_500') do
-          get '/mobile/v0/letters', headers: iam_headers
-          expect(response).to have_http_status(:internal_server_error)
-          expect(response.body).to match_json_schema('evss_errors')
+          # match_json_schema also uses the read method of File objects so we need to revert the stub being done above
+          allow(File).to receive(:read).and_call_original
+          expect(response.body).to match_json_schema('letters')
         end
       end
     end
   end
 
   describe 'GET /mobile/v0/letters/beneficiary' do
-    context 'with a valid veteran response' do
-      it 'matches the letter beneficiary schema' do
-        VCR.use_cassette('evss/letters/beneficiary_veteran') do
+    context 'with a valid lighthouse response' do
+      it 'matches the letters beneficiary schema' do
```

Comment:
Let's also add a test for the response body contents, similar to line 74 above for the letters endpoint. Schema tests don't really confirm that the data is correct.